examples/rosettacode/semordnilap.nit

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env nit
+#
+# This file is part of NIT ( http://www.nitlanguage.org ).
+# This program is public domain
+
+# Simple Rosettacode task for the detection of semordnilaps in a text file
```

Comment:
link the webpage, easier to understand what is the task (and what is a semordnilaps)